README.md

```diff
@@ -0,0 +1,72 @@
+# Budget App
+
+> An application built using Ruby on Rails to manage the expenses of any user. It allows the user the create categories under which are various transactions for the category. 
+
+## Screenshot
+| Categories Page |   Transactions page |
+| ---------- | ----------- |
+ | ![CategoriesPage](img/1.png) | ![TransactionsPage](img/2.png) |
+</p>
+
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md) for this project, " `The project should be deployed and accessible online.` " , please kindly deploy the application using Heroku, Netlify or GitHub Pages.

Reply:
@Theophileaseh, as requested by the previous reviewer, please make the application online accessible by deploying it on any platform you like as it is a part of the requirement. Kindly do so and then submit for a re-review.